### DISCUSSION TITLE
How can we extract entities from strings, can we have some examples?

### DISCUSSION THREAD

**Discussion by samdeesh** [nlp-compromise/nlp_compromise#914] [Q&A] [answered]
Example for entities:
String: `How are you doing today?`
Entities: You, today

`I sent you money last evening around 5PM`
entities: I, You, Money, last evening, 5 PM 

`Your account XX1234 has been credited with $10 at 10 am 20 April 2022 from Netflix refund and the total balance is USD  110`
entities:  XX1234, $10,  10 am 20 April 2022, Netflix,  USD  110

**Comment by spencermountain** [comment, marked as answer]
hi, this is covered in the docs, here
https://observablehq.com/@spencermountain/topics-named-entity-recognition

looks like you also would dates and numbers:
```js
let m = doc.topics()
m = m.concat(doc.numbers())
m = m.concat(doc.dates())
//and so forth..
m.debug()
```
cheers

**Reply by samdeesh** [reply]
just the [documentation page](https://github.com/spencermountain/compromise) has an incorrect link then, for [Named-Entities](https://observablehq.com/@spencermountain/compromise-topics)
Please check that. Also, for nodejs beginners, can you write one  complete working example if it is not too much to ask
